exercises/go-counting/canonical-data.json

```diff
@@ -0,0 +1,172 @@
+{
+  "exercise": "go-counting",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Test for black territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+		    "  B  ",
+		    " B B ",
+		    "B W B",
+		    " W W ",
+		    "  W  "
+		],
+        "x": 0,
+        "y": 1
+      },
+      "expected": {
+        "stone": BLACK
+        "territory": [(0, 0), (0, 1), (1, 0)]
```

Comment:
I think this wants to be:
```JSON
"territory": [[0,0], [0,1], [1,0]]
```
Are you trying to portray an array of tuples?  I am not sure if it is really possible to convey a tuple in JSON.